guide/how-it-works/ecash/overview.md

```diff
@@ -0,0 +1,222 @@
+
+## How ecash on bitcoin works
+Ecash on Bitcoin enhances privacy and security by using tokens for transactions. This section explains how you can create, use, receive, and convert ecash back to Bitcoin. You'll learn how protocols like Cashu and Fedimint make transactions more private and reduce onboarding friction, improving the overall user experience of transacting in Bitcoin.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/how-ecash-works@1x.png"
+   retina = "/assets/images/guide/how-it-works/ecash/how-ecash-works@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/how-ecash-works@2x.png"
+   mobileRetina = "assets/images/guide/how-it-works/ecash/how-ecash-works@2x.png"
+   alt-text = "An illustration of how users, ecash, and ecash mints interact."
+   caption = "How users interact with ecash mints and the lightning network."
+   width = 800
+   height = 456
+   layout = "full-width"
+%}
+
+Ecash on Bitcoin works by using protocols like Fedimint and Cashu, which both implement blinded signatures to ensure privacy and security. Users deposit Bitcoin, and in return, they receive ecash tokens that are backed by these deposits. The blinded signatures allow the tokens to be issued without revealing transaction details, ensuring strong privacy guarantees. Users can then spend and receive these ecash tokens, which remain anonymous and unlinkable to their identity, enabling private transactions within the Bitcoin ecosystem.
+
+## Bitcoin custody spectrum
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/custody-spectrum@1x.png"
+   retina = "/assets/images/guide/how-it-works/ecash/custody-spectrum@2x.png"
+   mobile = "/assets/images/guide/how-it-works/ecash/custody-spectrum@1x.png"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/custody-spectrum@2x.png"
+   alt-text = "An illustration of the spectrum of bitcoin custody."
+   caption = "Bitcoin custody spectrum"
+   width = 800
+   height = 456b
+   layout = "full-width"
+%}
+
+{% include tip/open.html color="blue" icon="info" label="Ecash is a custodial solution" %}
+
+Bitcoin-backed ecash is custodial, meaning someone else holds your Bitcoin. Be aware that this subjects useres to all custodial risks.
+
+{% include tip/close.html %}
+
+The spectrum of decentralization and custody for Bitcoin ranges from highly decentralized to highly centralized. Holding self-custodial Bitcoin on the base layer is the least risky and most decentralized, rated at 0. Self-custodial Lightning is slightly more centralized at 2, while custodial Lightning moves further towards centralization at 7. ETFs sit at the extreme end, being the most centralized and risky. Ecash offers improvements over custodial Lightning by providing better privacy and security.
+
+### Ecash vs Custodial Lightning
```

Comment:
Using a table like this doesn't really work on mobile (visually, I mean). Maybe you can just remove the explanation text from the table, so that readers can quickly parse it and get a sense of the comparison. After the table you could then use regular headings and the explanation text to explain the table above.

Reply:
I'll give this a try!

Reply:
I updated the table but it's in the introduction page now. I removed the explanation as a column from the table. It's now in a paragraph underneath it. I think this helps makes the table more manageable and mobile friendly. I wish the table was not full width of the page and instead just the same width as everything else, but I can't see to find a way to update the size.